File: pios.projekt.API/pios.projekt.models/Models/Exam.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace pios.projekt.models.Models
{
    public class Exam
    {
        public int Id { get; set; }
		public string Name { get; set; }
		public List<Student> Students { get; set; } = new List<Student>();
        public Teacher Teacher { get; set; }
		public Subject Subject { get; set; }
		public DateTime Date { get; set; } = new DateTime();
        public List<Question> Questions { get; set; } = new List<Question>();

    }
}

File: pios.projekt.front/src/Models/Exam.ts
import { Student } from './Student';
import { Teacher, TeacherClass } from './Teacher';
import { Subject, SubjectClass } from './Subject';
import { Question } from './Question';

export interface Exam {
    id: number;
    name: string;
    students: Student[];
    teacher: Teacher;
    subject: Subject;
    date: Date;
    questions: Question[];
}

export class ExamClass implements Exam {
    id = 0;
    name = '';
    students = [];
    subject = new SubjectClass();
    teacher = new TeacherClass();
    date = new Date();
    questions = [];
}
